basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import java.util.Random;
+import java.util.Arrays;
+
+public class Library {
+//    public boolean someLibraryMethod() {
+//        return true;
+//    }
+    public static void main(String[] args) {
+//        System.out.println(Arrays.toString(roll(6)));
+
+        int[] dupArr = new int[]{1,2,3,4,5,6,7,8,9,10,1};
+//        System.out.println(containsDuplicates(dupArr));
+
+        int[] avgArr = new int[]{1,2,3,6,6};
+//        System.out.println(averageOfArray(avgArr));
+
+        int[][] weeklyMonthTemperatures = {
+                {66, 64, 58, 65, 71, 57, 60},
+                {57, 65, 65, 70, 72, 65, 51},
+                {55, 54, 60, 53, 59, 57, 61},
+                {65, 56, 55, 52, 55, 62, 57}
+        };
+//        System.out.println(averageOfArrayOfArrays(weeklyMonthTemperatures));
+    }
+
+    public static int[] roll(int n) {
+        int min = 1;
+        int max = 6;
```

Comment:
I love that you are defining a min and a max instead of just throwing magic numbers into the random method